#From: Dieter Miosga  <dmiosga6200@gmail.com>
#Date: Thu, 15 Jan 2015 00:52:59 -0000
#
#
#  The AMD FX-6100 Bulldozer/Vishera processor does perfectly support 
#  3dnow instructions by its 3dnowprefetch feature. 
#  Kernels compiled for the Barcelona/AMD-K10-family with explicit use of 3dnow instructions
#  in  both, kernel binary and modules, run without problems on FX-6100  systems. 
#
#  Inibit the die() call and change the messages content
#  Dieter Miosga, 2015-01-15 
#  
#  Signed-off-by: Zeusellito, the true father of gods <secure-piloting@freed-universe.com>
diff -uNr linux-3.18/arch/x86/boot/cpu.c linux-3.18.3-ossmga/arch/x86/boot/cpu.c
--- linux-3.18/arch/x86/boot/cpu.c	2014-12-07 22:21:05.000000000 +0000
+++ linux-3.18.3-ossmga/arch/x86/boot/cpu.c	2015-01-18 13:14:15.968113636 +0000
@@ -88,7 +88,7 @@
 	}
 
 	if (err_flags) {
-		puts("This kernel requires the following features "
+		puts("This booting kernel check detects requirement for following features "
 		     "not present on the CPU:\n");
 		show_cap_strs(err_flags);
 		putchar('\n');
diff -uNr linux-3.18/arch/x86/boot/main.c linux-3.18.3-ossmga/arch/x86/boot/main.c
--- linux-3.18/arch/x86/boot/main.c	2014-12-07 22:21:05.000000000 +0000
+++ linux-3.18.3-ossmga/arch/x86/boot/main.c	2015-01-18 13:16:21.336110408 +0000
@@ -147,9 +147,10 @@
 
 	/* Make sure we have all the proper CPU support */
 	if (validate_cpu()) {
-		puts("Unable to boot - please use a kernel appropriate "
-		     "for your CPU.\n");
-		die();
+		puts("If you did not use special compilation parameters, this is:\n"
+		     "Unable to boot - please use a kernel appropriate "
+		     "for your CPU,\n Check your kernel setup in case of any following kernel crash!!!");
+		/*die();*/
 	}
 
 	/* Tell the BIOS what CPU mode we intend to run in. */
